packages/odyssey-react-mui/src/MenuItem.tsx

```diff
@@ -10,38 +10,33 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
-import { memo, forwardRef } from "react";
+import { ReactNode, memo } from "react";
 import { MenuItem as MuiMenuItem } from "@mui/material";
 import { menuItemClasses } from "@mui/material/MenuItem";
-import type { MenuItemProps as MuiMenuItemProps } from "@mui/material";
 
-export interface MenuItemProps
-  extends Omit<
-    MuiMenuItemProps,
-    | "component"
-    | "dense"
-    | "disableGutters"
-    | "divider"
-    | "focusVisibleClassName"
-  > {
-  /**
-   * Toggles whether or not the MenuItem represents a destructive action.
-   */
+export type MenuItemProps = {
+  hasInitialFocus?: boolean;
+  children: ReactNode;
+  isSelected?: boolean;
   isDestructive?: boolean;
-}
+};
```

Comment:
I think we should add `onClick`.

But also, it might be good to search Atko and see how this is currently used:
```tsx
<MenuItem content={undefined} rel={undefined} rev={undefined}>
<MenuItem value={DEFAULT_OPTION_VALUE} data-testid="select-option">
<MenuItem key={index} onClick={() => navigateTo(link)}>
<MenuItem value="true">
<MenuItem onClick={logOut}>{t('topBar.dropdown.signOut')}</MenuItem>
```

There are all the variations I found from a quick search.

Reply:
I added the following props:

- `content`
- `data-testid`
- `href`
- `onClick`
- `rel`
- `rev`
- `value`

I don't know what `rev` is either, so maybe we don't need that one.

For `onClick`, I have an optional `onClick` function and the existing `closeMenu` behavior you'd previously added.

Reply:
First one has these as undefined as current odyssey version breaks typescript, as reported a little while back :)

In reality, it would only be using `children`.